lua/terrortown/lang/zh_hans.lua

```diff
@@ -37,15 +37,15 @@ L.win_time = "时间用尽，叛徒失败了。"
 L.win_traitors = "叛徒取得了胜利！"
 L.win_innocents = "叛徒们被击败了！"
 L.win_nones = "无人胜出！（平局）"
-L.win_showreport = "来看一下 {num} 秒的回合总结吧！"
+L.win_showreport = "一起来看一下 {num} 秒的回合总结吧！"
 
 L.limit_round = "达到回合限制。即将加载下一张地图。"
 L.limit_time = "达到时间限制。即将加载下一张地图。"
 L.limit_left = "新地图将在 {num} 回合或 {time} 分钟后切换。"
 
 -- Credit awards
-L.credit_all = "你的阵营因为表现获得了 {num} 点积分。"
-L.credit_kill = "你杀死 {role} 获得了 {num} 点积分。"
+L.credit_all = "你的阵营因为表现获得了 {num} 点信用点。"
```

Comment:
String that has been partially translated into '信用点' has been used back to the previous translation

Reply:
Not sure when that got in, but I'm of the opinion that 信用点 isn't a good translation because 信用 in this context stands for credibility, and doesn't really have anything to do with a spendable currency.

Reply:
That does :)